index.d.ts

```diff
@@ -46,9 +46,28 @@ declare namespace mapObject {
 		shouldRecurse?: boolean;
 	}
 
-	export type mapObjectSkip = symbol
+
+	/**
+	Return this value from a `mapper` function to remove a key from an object.
+
+	```js
+	const mapObject = require('map-obj');
+
+	const object = {one: 1, two: 2}
+	const mapper = (key, value) => value === 1 ? [key, value] : mapObject.mapObjectSkip
+	const result = mapObject(object, mapper);
+
+	console.log(result);
+	//=> {one: 1}
+	```
+	*/
+	const mapObjectSkip: typeof UniqueSymbol
 }
 
+// unique symbol cannot be declared in a namespace directly, so we declare it top-level
+// See: https://github.com/sindresorhus/map-obj/pull/38#discussion_r702396878
+declare const UniqueSymbol: unique symbol;
```

Comment:
It should still have a descriptive name and be camel-cased.

Reply:
camel cased via 358fafd. 

It's only used internally, and for that I think it's perfectly descriptive? Can you rename it yourself to whatever you prefer at this point?

Reply:
Even it's internally, it's good to have descriptive names. You don't call all your variables `const variable1 = …; const variable2 ...`. But it's. I can clean it up later.